profiles/base.yaml

```diff
@@ -82,6 +82,7 @@ buff_nonspells:
 dance_actions_stealth:
 thanatology:
 necromancer_healing:
```

Comment:
The setting name `heal_threshold` is rather generic. What would you think about putting `necro` or `necromancer` somewhere in the name?

Or maybe it should be a sub-setting of `thanatology` or `necromancer_healing`? @jandersson might have a better idea on that one...

Reply:
So I may add this to the regular empath healing logic as well, so not sure at that point if it should be a separate variable, or if all healing yaml should be rolled up..

Reply:
Maybe `self_heal_threshold`?

I'm concerned it could be confused with the existing `health_threshold` setting

Reply:
Hmm wound_level_threshold?

Reply:
I suppose making it a sub of necromancer_healing would be better.

Reply:
Moved under necromancer_healing, and renamed wound_level_threshold

Reply:
let me know if the new variable name and location work for you.